src/System.Private.CoreLib/shared/System/Diagnostics/ConditionalAttribute.cs

```diff
@@ -2,16 +2,18 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
+
```

Comment:
Nit: delete this blank line... I believe the goal is that once we've annotated every file, we turn on the setting for the whole project, and then just search/replace this one line in every file to remove it... when we do, we don't want extraneous blank lines left behind.